setup.py

```diff
@@ -2,7 +2,7 @@
 from setuptools import setup, find_packages
 
 install_requires = [
-    "PyYAML==3.11", "requests==2.20.1", "simplejson==3.8.2",
+    "PyYAML==5.1", "requests==2.20.1", "simplejson==3.8.2",
```

Comment:
Can you please update the link in requirements file as well.

Reply:
LOL, is this real life?